src/app/interfaces/user.ts

```diff
@@ -0,0 +1,2 @@
+export interface User {
+}
```

Comment:
Should we just delete this file if it's not being used? @BlakeDraper

Reply:
Your call. I created it to be the interface for the User, but then realized the STN services use the term 'member' (ew) within the DB. Could still be a use for the the 'User' construct within the frontend context. But you're right, it's not in use right now.